frameworks/pytorch/get_started_mnist_deploy.ipynb

```diff
@@ -6,23 +6,15 @@
    "source": [
```

Comment:
<p>Style nit: consistently use double quotes - I see there's <code>'rb'</code> in markdown in the notebook, and also in the inference.py script. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>